src/main/java/com/homework/springboot/service/SchedulerService.java

```diff
@@ -0,0 +1,44 @@
+package com.homework.springboot.service;
+
+import com.homework.springboot.model.User;
+
+import com.homework.springboot.service.impl.WRuntimeException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SchedulerService {
+
+    private final UserService userService;
+
+    private final EmailService emailService;
+
+    @Scheduled(cron="${cronExpression}")
+    public void sendMailToUsers() throws WRuntimeException {
+        LocalDate date = LocalDate.now();
+        int month = date.getMonthValue();
+        int day = date.getDayOfMonth();
+        List<User> list = userService.getUsersByBirthday(month, day);
+        if (!list.isEmpty()) {
```

Comment:
Did you need this check? You did not answer or change the code, just marked them as resolved.
Please add an explanation.